src/test/ui/anon-params-edition-hygiene.rs

```diff
@@ -0,0 +1,10 @@
+// check-pass
```

Comment:
r=me with the test (and related tests, i.e. `src/test/ui/anon-params*`) moved into a folder `src/test/ui/anon-params/`.